infra/modules/leader/main.tf

```diff
@@ -11,7 +11,7 @@ resource "google_cloud_run_v2_service" "leader" {
 
     vpc_access {
       connector = var.connector_id
-      egress    = "PRIVATE_RANGES_ONLY"
+      egress    = "ALL_TRAFFIC"
```

Comment:
do we still want this?

Reply:
yes, since we want all traffic to leave our VPC.